executor/runtime/types/sidecars.go

```diff
@@ -183,6 +193,10 @@ func shouldStartTitusStorage(cfg *config.Config, c Container) bool {
 	return c.EBSInfo().VolumeID != ""
 }
 
+func shouldStartContainerTools(cfg *config.Config, c Container) bool {
+	return cfg.ContainerToolsImage != ""
```

Comment:
The idea is that eventually every container will get this, correct?

Reply:
Yep.

Reply:
But I figured we'll leave this in here, in an emergency we just set the FP to `""` and no container tools

Reply:
Makes sense, thanks!